packages/squiggle-lang/__tests__/Reducer/Reducer_Type/Reducer_Type_TypeChecker_arguments_test.res

```diff
@@ -1,42 +1,41 @@
-module Bindings = Reducer_Bindings
-module ErrorValue = Reducer_ErrorValue
-module Expression = Reducer_Expression
-module ExpressionT = Reducer_Expression_T
-module InternalExpressionValue = ReducerInterface_InternalExpressionValue
-module ProjectAccessorsT = ReducerProject_ProjectAccessors_T
-module T = Reducer_Type_T
-module TypeChecker = Reducer_Type_TypeChecker
+// module Bindings = Reducer_Bindings
+// module ErrorValue = Reducer_ErrorValue
+// module Expression = Reducer_Expression
+// module ProjectAccessorsT = ReducerProject_ProjectAccessors_T
+// module T = Reducer_Type_T
+// module TypeChecker = Reducer_Type_TypeChecker
 
-open Jest
-open Expect
+// open Jest
+// open Expect
 
-let checkArgumentsSourceCode = (aTypeSourceCode: string, sourceCode: string): result<
-  'v,
-  ErrorValue.t,
-> => {
-  let reducerFn = Expression.reduceExpressionInProject
-  let rResult =
-    Expression.BackCompatible.parse(sourceCode)->Belt.Result.map(expr =>
-      reducerFn(expr, Bindings.emptyBindings, ProjectAccessorsT.identityAccessors)
-    )
-  rResult->Belt.Result.flatMap(result =>
-    switch result {
-    | IEvArray(args) => TypeChecker.checkArguments(aTypeSourceCode, args, reducerFn)
-    | _ => Js.Exn.raiseError("Arguments has to be an array")
-    }
-  )
-}
+// let checkArgumentsSourceCode = (aTypeSourceCode: string, sourceCode: string): result<
+//   'v,
+//   ErrorValue.t,
+// > => {
+//   let reducerFn = Expression.reduceExpressionInProject
+//   let rResult =
+//     Expression.BackCompatible.parse(sourceCode)->Belt.Result.map(expr =>
+//       reducerFn(expr, Bindings.emptyBindings, ProjectAccessorsT.identityAccessors)
+//     )
+//   rResult->Belt.Result.flatMap(result =>
+//     switch result {
+//     | IEvArray(args) => TypeChecker.checkArguments(aTypeSourceCode, args, reducerFn)
+//     | _ => Js.Exn.raiseError("Arguments has to be an array")
+//     }
+//   )
+// }
```

Comment:
more comments

Reply:
Ok, I read type-related code some more and decided to delete it all. It'll probably will be easier to redo from scratch, and the old files are still in repo for reference.